### DISCUSSION TITLE
cannot set up my ESP 32 cam - issues with initial set up

### DISCUSSION THREAD

**Discussion by markusruebe** [jomjol/AI-on-the-edge-device#2616] [Q&A]
I have massive problems to configure my use P 32 camera for my water measurement. The device is very slow. The web GUI is not responding.
I also have problems with the configuration of ROI's and the two reference images.
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/53835421/18a5faba-f688-4003-a94c-2c0f48a907d1)
hey, see you see in the image I have "m3" as a reference but not the other image from the set up. Can I also add reference images on my own?

**Comment by caco3** [comment]
First make sure your wifi is work good.
Check the RSSI, try it once close to your accesspoint.
Try to use an external antenna.

And read the documentation: https://jomjol.github.io/AI-on-the-edge-device-docs
